sw/vendor/cryptoc/OWNERS

```diff
@@ -0,0 +1,3 @@
+mnissler@chromium.org
```

Comment:
I'm a little amused that we're vendoring in an OWNERS file. =P

Reply:
Should I remove the OWNERS file?

Reply:
No, not worth adding patches just to do a cosmetic change.

Reply:
isn't there a way to exclude specific files during the vendor process?